docs/platforms/apple/common/install/direct-install.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Direct Install 
+description: "Integrate Sentry into your Xcode project by using our pre-compiled frameworks."
+---
+
+To integrate Sentry into your Xcode project, follow these steps:
+
+1. Download the latest version of the SDK from the Sentry Cocoa [Releases page](https://github.com/getsentry/sentry-cocoa/releases).
+2. Each release contains the following four framework options:
+    - Sentry-Dynamic.xcframework.zip: Dynamic framework
+	- Sentry.xcframework.zip: Static framework
+	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
+	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Framework without UIKit or AppKit linking and related features
```

Comment:
`m`: Should we also mention that these are static for consistency?

Reply:
```suggestion
	- SentrySwiftUI.xcframework.zip: Framework with SwiftUI support
	- Sentry-WithoutUIKitOrAppKit.xcframework.zip: Static framework without UIKit or AppKit linking and related features
```